shared/devices/row.tsx

```diff
@@ -12,19 +12,19 @@ type OwnProps = {
 
 export const NewContext = React.createContext(new Set())
 
-export default (ownProps: OwnProps) => {
+const Container = (ownProps: OwnProps) => {
   const {deviceID, firstItem} = ownProps
   const device = C.useDevicesState(s => s.deviceMap.get(deviceID))
-  if (!device) return null
```

Comment:
not sure why lint wasn't picking this up before but lint is correct, conditional hooks below